### PR TITLE
Add support for the 'edited_message' telegram event

### PR DESCRIPTION
**Pull request by repkam09**
Small change to add support for the 'edited_message' telegram event so if someone on the Telegram side edits a message, that edit is then sent to the IRC side. Before, the edit would happen on the Telegram side (as usual) but the change would not be sent to the IRC side.

To support this change, the node telegram bot npm package was bumped to the latest version which supports this new event.

### CODE REVIEW

File: lib/libteleirc.js [hunk 1]
```diff
@@ -208,11 +208,12 @@ class TeleIrc {
         this.tgRateLimiter = new MessageRateLimiter(
             this.config.tg.maxMessagesPerMinute,
             60,
-            function(message) {
+            function (message) {
```
Comment (robbyoconnor) [inline lib/libteleirc.js]: Use arrow functions

Comment (robbyoconnor) [inline lib/libteleirc.js]: It's much cleaner here.

**Comment by robbyoconnor** [comment]
you are going to need to rebase after #26 is merged...one of us will have to..

**Comment by jflory7** [comment]
@repkam09 Would you mind rebasing so we can merge this in? :) Also, any idea why your PR went straight to the `master` branch instead of `devel`?

**Comment by robbyoconnor** [comment]
I caused the conflict by adding `dotenv` -- sorry :)

**Comment by thenaterhood** [comment]
Does this also add support for passing files to IRC? The end of the diff looks like it does.

**Comment by jflory7** [comment]
Whoops. Any reason why we can't merge this into the repository? Seems like we're all good?

**Comment by robbyoconnor** [comment]
After it's squashed because I'm OCD about revision history @jflory7

**Comment by robbyoconnor** [comment]
I think you can squash it when you merge it actually...and probably should.

**Comment by thenaterhood** [comment]
GitHub pulls can do squash and merge. That's generally preferred when
merging into a dev branch (but not to master/prod; causes headaches).

**Comment by robbyoconnor** [comment]
You may wanna rebase to `devel`?

**Comment by jflory7** [comment]
@robbyoconnor Yeah, I noticed that it went to `master` and I was working on rebasing locally, but have some merge conflicts I needed help on solving. I shared more info in our Telegram group.

**Pull request merged**